include/hpp/manipulation/graph/edge.hh

```diff
@@ -163,11 +166,7 @@ namespace hpp {
           /// Constructor
           Edge (const std::string& name);
 
-          /// Constraint to project a path.
-          /// \return The initialized constraint.
-          ConstraintSetPtr_t pathConstraint() const;
-
-          virtual ConstraintSetPtr_t buildConfigConstraint();
+	  virtual ConstraintSetPtr_t buildConfigConstraint();
```

Comment:
Could you configure your editor so as to avoid white space changes ?
The change on `buildConfigConstraint` should not be here.

Reply:
I meant that I would like you to amend the commit.